package.json

```diff
@@ -1,14 +1,20 @@
 {
-  "name": "cast-monorepo",
+  "name": "cast",
   "version": "0.1.0",
-  "description": "A monorepo for the Cast app",
-  "private": true,
+  "description": "A podcast listening PWA",
   "scripts": {
-    "preinstall": "npx only-allow pnpm",
-    "build:presentation": "cd presentation && pnpm build",
-    "build:app": "cd app && pnpm build",
-    "lint": "prettier --check . --plugin-search-dir=.",
-    "fix": "prettier --write . --plugin-search-dir=."
+    "dev": "svelte-kit dev",
+    "build": "svelte-kit build",
+    "package": "svelte-kit package",
+    "preview": "svelte-kit preview",
+    "check": "svelte-check --tsconfig ./tsconfig.json",
+    "check:watch": "svelte-check --tsconfig ./tsconfig.json --watch",
+    "eslint": "eslint --ignore-path .gitignore .",
+    "fix": "eslint --ignore-path .gitignore . --fix",
+    "fix_": "prettier --write . --plugin-search-dir=.",
```

Comment:
Not sure how to merge these `fix` and `lint`. Need help.

Reply:
For `lint`, the command will be: `eslint --ignore-path .gitignore .; prettier --check . --plugin-search-dir=.; svelte-check --tsconfig ./tsconfig.json`. For `fix`, ESLint and Prettier should run in fixing mode. The `check` and `check:watch` can be removed.

P.S. While you're at it, pass `--cache` to the ESLint commands. It will make them run faster :) But add `.eslintcache` to Git ignore though

Reply:
For some reason upon running `lint` and `fix` I received `ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  not found: eslint;` and `Invalid option '--write' - perhaps you meant '--rule'`. What am I doing wrong?

Reply:
Don't know what's wrong here exactly, but I'd try removing the `eslint` and `prettier` scripts and inlining their commands